manager/integration/tests/test_node.py

```diff
@@ -350,6 +350,23 @@ def test_replica_scheduler_no_disks(client):  # NOQA
     common.wait_for_volume_delete(client, vol_name)
 
 
+@pytest.mark.node  # NOQA
+def test_disable_scheduling_on_cordoned_node(client):  # NOQA
```

Comment:
Add `@pytest.mark.skip(reason="TODO")`, so we know which tests need to be implemented.

Reply:
Thanks for the input.